gammapy/data/observations.py

```diff
@@ -0,0 +1,569 @@
+from ..utils.table import table_row_to_dict
+from ..utils.time import time_ref_from_dict
+
+__all__ = [
+    'ObservationIACT',
+    'DataStoreObservationIACT',
+    'ObservationList',
+]
+
+log = logging.getLogger(__name__)
+
+
+class ObservationIACT(object):
+    """Container class for an IACT observation
+
+    Parameters follow loosely the "Required columns" here:
+    http://gamma-astro-data-formats.readthedocs.io/en/latest/data_storage/obs_index/index.html#required-columns
+
+    Parameters
+    ----------
+    obs_id : int
+        Observation ID
+    gti : `~gammapy.data.GTI`
+        Good Time Intervals, see: http://gamma-astro-data-formats.readthedocs.io/en/latest/events/gti.html
+    events : `~gammapy.data.EventList`
+        Event list, see: http://gamma-astro-data-formats.readthedocs.io/en/latest/events/events.html
+    aeff : `~gammapy.irf.EffectiveAreaTable2D`
+        Effective area, see: http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/aeff/index.html
+    edisp : `~gammapy.irf.EnergyDispersion2D`
+        Energy dispersion, see: http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/edisp/index.html
+    psf : `~gammapy.irf.PSF3D` or `~gammapy.irf.EnergyDependentMultiGaussPSF` or `~gammapy.irf.PSFKing`
+        Tabled Point Spread Function, see: http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/psf/index.html
+    bkg : `~gammapy.irf.Background2D` or `~gammapy.irf.Background3D`
+        Background rate, see: http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/bkg/index.html
+    pointing_radec : `~astropy.coordinates.SkyCoord`
+        Pointing RA / DEC sky coordinates
+    pointing_altaz : `~astropy.coordinates.SkyCoord`
+        Pointing ALT / AZ sky coordinates
+    pointing_zen : `~astropy.coordinates.SkyCoord`
+        Pointing zenith angle sky
+    observation_time_duration : `~astropy.units.Quantity`
+        Observation time duration in seconds
+
+        The wall time, including dead-time.
+    observation_live_time_duration : `~astropy.units.Quantity`
+        Live-time duration in seconds
+
+        The dead-time-corrected observation time.
+
+        Computed as ``t_live = t_observation * (1 - f_dead)``
+        where ``f_dead`` is the dead-time fraction.
+    observation_dead_time_fraction : `float`
+        Dead-time fraction
+
+        Defined as dead-time over observation time.
+
+        Dead-time is defined as the time during the observation
+        where the detector did not record events:
+        https://en.wikipedia.org/wiki/Dead_time
+        https://adsabs.harvard.edu/abs/2004APh....22..285F
+
+        The dead-time fraction is used in the live-time computation,
+        which in turn is used in the exposure and flux computation.
+    tstart : `~astropy.units.Quantity`
+        Observation start time
+    tstop : `~astropy.units.Quantity`
+        Observation stop time
+    telescope_ids : list of int
```

Comment:
Remove telescope_ids, not used at the science tool level.

If we want to preserve "provenance" info, it can be in `meta`.
Let's aim to have the obs interface small and only with the pieces of info we need.

Reply:
Done